examples/01-guest-shopper-auth-token.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Get authorization token for a guest user
+ * Usage: ts-node examples/01-guest-shopper-auth-token.ts
+ * Note: Replace configuration parameters before running
+ */
+import { helpers, ClientConfig } from "../dist";
```

Comment:
```suggestion
import { helpers, ClientConfig } from "@commerce-sdk";
```
Importing from `dist` requires being in this repo, installing everything, and running the build process.

Importing from `commerce-sdk` works anywhere and only requires installing `commerce-sdk`.

Reply:
Referring/Importing the same repo gives an error.  Since examples are in the repo, I think this is easier  to run